page_objects/login_page.js

```diff
@@ -1,31 +1,62 @@
 // const { element } = require("protractor");
+const random = require("../node_modules/random-js").Random;
+
 let Input = require("../elements/input.element")
 let Button = require("../elements/button.element")
 let TextView = require("../elements/text.view")
 
-let BasePage = require ("../base/base.page")
+let BasePage = require("../base/base.page")
 
 let emailLocator = by.css('#email');
 let passwordLocator = by.css('#passwd');
 let submitbtnLocator = by.css('#SubmitLogin');
 let textloginLocator = by.css('#login_form>.page-subheading');
 let errorpassword = by.css('.alert li')
+let emailforRegLocator = by.css('#email_create')
+//let textPersonalInfLocator = by.xPath('//*[.=`Your personal information`]')
 
-class LoginPage extends BasePage {   
-    async login (email,password) {
+class LoginPage extends BasePage {
+    async login(email, password) {
         await allure.createStep(`Logging in ${email} / ${password}`, async () => {
             await this.getEmailInput().sendKeys(email);
             await this.getPassInput().sendKeys(password);
             await this.getSubmButInput().click();
         })()
     }
-    async getErrorMessage () {
-        return await this.getErrorElement().getText(); 
-    }   
-    async clearCredentials () {
-         await this.getEmailInput().clear();
-         await this.getPassInput().clear();
+
+
+    getRandomEmail() {
+        return this.getRandomString(12);
+    }
+
+    getRandomString(characterLength) {
```

Comment:
1. use randomstring library, as we did, https://www.npmjs.com/package/randomstring
2. the method is not related to the page object functionality, please create class Utils and move it there
3. if method name `getRandomString` it should return string, not email

Reply:
fixed